assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -444,6 +444,29 @@ const DeckLayoutFooter: React.FC = () => {
     }
   }, [lastCheckResults, isPreviewMode]);
 
+  const isActivityHistoryTimeStampIsZero = () => {
+    const trackingStampKey = `session.visitTimestamps.${currentActivity?.id}`;
+    const globalSnapshot = getEnvState(defaultGlobalEnv);
```

Comment:
I guess this works, but aren't there methods also for just getting a single value?